README.md

```diff
@@ -10,11 +10,15 @@ Supports the latest CSS syntax and features, including custom properties, range
 $ npm install stylelint
 ```
 
-_Note: see issue [#133](https://github.com/stylelint/stylelint/issues/133) for details of our progress towards more accurate line numbers_.
+_The reported line numbers of some warnings may be inaccurate. See issue [#133](https://github.com/stylelint/stylelint/issues/133) for details of our progress towards resolving this._
```

Comment:
We could be specific: "Currently, reported line numbers are only *approximate*, marking the beginning of the CSS node to which the warning relates."